Xamarin.Forms.Platform.Android/ResourceManager.cs

```diff
@@ -24,6 +24,11 @@ internal static Drawable GetFormsDrawable(this Resources resource, FileImageSour
 			if(drawable == null)
 			{
 				var bitmap = GetBitmap(resource, file) ?? BitmapFactory.DecodeFile(file);
+				if (bitmap == null)
+				{
+					var source = Registrar.Registered.GetHandler<IImageSourceHandler>(fileImageSource.GetType());
+					bitmap = source.LoadImageAsync(fileImageSource, Forms.Context).GetAwaiter().GetResult();
+				}
```

Comment:
won't this lock the UI thread ?

Reply:
Less than GetBitmap/DecodeFile.
It is coded like this on iOS already. 
This would make Android as extensible as iOS.